features/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/rum/internal/instrumentation/gestures/GesturesListenerScrollSwipeTest.kt

```diff
@@ -279,6 +280,73 @@ internal class GesturesListenerScrollSwipeTest : AbstractGesturesListenerTest()
         verifyNoMoreInteractions(rumMonitor.mockInstance)
     }
 
+    @ParameterizedTest
+    @ValueSource(
+        strings = [
+            GesturesListener.SCROLL_DIRECTION_DOWN,
+            GesturesListener.SCROLL_DIRECTION_UP,
+            GesturesListener.SCROLL_DIRECTION_LEFT,
+            GesturesListener.SCROLL_DIRECTION_RIGHT
+        ]
+    )
+    fun `M send a tap rum event if target is a non scrollabld`(
```

Comment:
typo here, should be `scrollable` (I don't have suggestion button available, because reviewing individual commit)